BUILD.bazel

```diff
@@ -44,14 +47,14 @@ cmake(
 )
 
 cmake(
-    name = "libcurl_openssl",
+    name = "libcurl_boringssl",
     cache_entries = {
         "CMAKE_BUILD_TYPE": "Release",
         "BUILD_SHARED_LIBS": "OFF",
         "BUILD_TESTING": "OFF",
-        "CMAKE_USE_OPENSSL": "TRUE",
-        "CURL_DISABLE_LDAP" : "TRUE",
-        "OPENSSL_ROOT_DIR": "$$EXT_BUILD_DEPS$$/openssl",
+        "CMAKE_USE_LIBSSH": "FALSE",
+        "CMAKE_USE_LIBSSH2": "FALSE",
+        "CMAKE_USE_WOLFSSL": "FALSE",
```

Comment:
Let's leave a comment / breadcrumb that explains that we needed to turn these off because at least on macOS if libssh or libssh2 happens to be installed then cmake will attempt to link against the library but Bazel will expect all the symbols to be available and then the binary will crash when it attempts to dynamically find the symbols at runtime. Thanks!

Reply:
Please also add a comment above where you building without openssl but still are disabling libssh. It would be even better if we could factor out the `cache_entries` that are the same for both `cmake()` targets and then just have the comment in one place.

Reply:
Done!